src/pip/_internal/operations/prepare.py

```diff
@@ -131,12 +132,15 @@ def _download_url(
     else:
         url = link.url_without_fragment
 
+    redacted_url = redact_auth_from_url(url)
+
     if total_length:
-        logger.info("Downloading %s (%s)", url, format_size(total_length))
+        logger.info("Downloading %s (%s)", redacted_url,
+                    format_size(total_length))
```

Comment:
I would format like

```python
        logger.info(
            "Downloading %s (%s)", redacted_url, format_size(total_length)
        )
```

to match the rest of this file.

Reply:
Sure thing :) 
Thanks!